src/main/java/pl/lodz/p/edu/flanki/entities/EventOwner.java

```diff
@@ -0,0 +1,29 @@
+package pl.lodz.p.edu.flanki.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@Builder
+@Entity
+@Table(name = "EVENT_OWNER")
+@NoArgsConstructor
+@AllArgsConstructor
+public class EventOwner {
```

Comment:
Is it good idea to create such abstraction? I would say, that EventOwner is the same as User (they have the same id). As I offered in database diagram, maybe we could create relation many to many between User and and Event? What do you think about it?